src/bundle/Resources/public/js/scripts/fieldType/ezboolean.js

```diff
@@ -21,19 +21,6 @@
                 errorMessage,
             };
         }
-
-        /**
-         * Updates the state of checkbox indicator.
-         *
-         * @method updateState
-         * @param {Event} event
-         * @memberof EzBooleanValidator
-         */
-        updateState(event) {
```

Comment:
If you are removing the method please remove also the event from eventsMap. This will throw an error.